bin/strata-client/src/main.rs

```diff
@@ -408,7 +409,7 @@ fn start_sequencer_tasks(
         engine,
         bitcoin_client,
         ..
```

Comment:
Don't really like how this is being cloned but I do like the idea of passing it to the thing that uses it on 449.

Could this be reworked to repeat the pattern with the other launcher fns?